.github/workflows/scripts/start-kind.sh

```diff
@@ -155,6 +155,7 @@ WorkerChartOptions:
   Chart: kubeslice-worker
   Repo: "https://raw.githubusercontent.com/kubeslice/dev-charts/gh-pages/"
   Namespace: kubeslice-system
+  Version: 0.4.6
```

Comment:
this will test against worker version 0.4.6 and not against current PR

Reply:
using the latest version reverted the change